lab_3_ann_retriever/main.py

```diff
@@ -194,6 +254,13 @@ def save(self, file_path: str) -> bool:
         Returns:
             bool: True if saved successfully, False in other case
         """
+        if not file_path or not isinstance(file_path, str):
+            return False
+        with open(file_path, 'w', encoding='utf-8') as file:
```

Comment:
json files contains unicode - try to save with russian letters

Reply:
done